[WIP]AccountImportMethod screen

e-molochko opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ozalexo/chronomint-rn/18)
<!-- Reviewable:end -->

e-molochko committed:
Added AccountImportMethod screen

e-molochko committed:
Added AccountImportMethod screen

e-molochko committed:
Added NavHeader component

e-molochko committed:
Added NavHeader component

ozalexo commented on src/screens/AccountImportMethod/AccountImportMethod.js:
```diff
@@ -0,0 +1,94 @@
+  onSelectAccountImportMethod: PropTypes.func,
+}
+
+class Method extends PureComponent {
```
I suggest to move this class into `src/screens/AccountImportMethod/Method.js` (yes, right here, because it would not reused in other parts of code).

ozalexo commented on src/screens/AccountImportMethod/AccountImportMethod.js:
```diff
@@ -0,0 +1,94 @@
+import TextButton from '../../components/TextButton'
+import styles from './AccountImportMethodStyles'
+
+export default class SelectAccountImportMethod extends PureComponent {
```
Name of exported class must be the same as file name. Also I suggest to rename "AccountImportMethod" to "ImportMethod", because in this case our screens' names will be the same as on prototype (easy to search files)

e-molochko committed:
Deleted custom header
- Added importmethod screen to navigation and storybook
- Added wrapper for screens at storybook
- Fixed styles

e-molochko committed:
Deleted custom header
- Added importmethod screen to navigation and storybook
- Added wrapper for screens at storybook
- Fixed styles

e-molochko committed:
Added EnterPrivateKey screen
- Also added EnterPrivateKey screen to storybook

e-molochko committed:
Added EnterPrivateKey screen
- Also added EnterPrivateKey screen to storybook

ozalexo commented on src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js:
```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native'
+import {Header} from 'react-navigation'
```
According to the information from official react-navigation web site, this is temporary "solution". So, we should not use Header.HEIGHT.

ozalexo commented on src/screens/EnterPrivateKey/EnterPrivateKeyStyles.js:
```diff
@@ -0,0 +1,11 @@
+export default StyleSheet.create({
+  screenView: {
+    alignSelf: 'center',
+    width: '80%',
```
This is incorrect. Width must be 100% and please implement it via `flex` and `flexDirection` with no 'width' in styles.

ozalexo commented on src/screens/ImportMethod/ImportMethod.js:
```diff
@@ -0,0 +1,87 @@
+  View,
+} from 'react-native'
+import textConstants from '../../locales/en'
+import PropTypes from 'prop-types'
```
Code style:
All imports with absolute paths must be placed above all other imports with relative paths ('../')

ozalexo commented on src/screens/ImportMethod/ImportMethod.js:
```diff
@@ -0,0 +1,87 @@
+
+  render() {
+    const {
+      ImportMethods,
```
Code style:
all variables must be named in camel-case and starts with small character. In this case it must be importMethods.
But even better to call it `importMethodList` (with no 's' in the middle)

ozalexo committed:
[RFR] Added Start screen (#17)

* CM-151 iOS: Added SafeAreaView for iPhone X

* CM-152 Fixed drawers and Header

* Added icons and SplashScreen for Android

* CM-222 StartScreen

* Cm-222 Fixed Android keyboard vertical offset

* CM-222 Applied CR comments

e-molochko committed:
Resolving conflicts and PR comments changes
